lightly/cli/upload_cli.py

```diff
@@ -40,8 +40,12 @@ def _upload_cli(cfg, is_cli_call=True):
         print_as_warning('Please specify your access token.')
         cli_api_args_wrong = True
 
-    dataset_id_ok = dataset_id and len(dataset_id) > 0
-    new_dataset_name_ok = new_dataset_name and len(new_dataset_name) > 0
+    if dataset_id and len(dataset_id) > 0:
```

Comment:
if `dataset_id` is a string this is actually redundant

Reply:
Yes, it also was before. I'll change it.